Makefile

```diff
@@ -194,7 +194,6 @@ test-gear: # Crates except gclient, gcli, gsdk are excluded to significantly dec
 		--exclude gclient \
 		--exclude gcli \
 		--exclude gsdk \
-		--exclude gear-authorship \
```

Comment:
they're still pretty long and heavy for pre-commit, isn't it?

Reply:
On par with some prop tests. About 40-ish seconds

Reply:
If it's not too important for pre-commit then maybe we can keep excluding it. After all, changes in block production do not come that often comparing to runtime